docs/docs/pages/style-spec/other.md

```diff
@@ -0,0 +1,385 @@
+---
+title: Other
+id: other
+description: The value for any layout or paint property may be specified as a function.
+contentType: specification
+order: 11
+layout: page
+hideFeedback: true
+products:
+- MapLibre Style Specification
+prependJs:
+    - "import SDKSupportTable from '../../components/sdk_support_table';"
+    - "import ref from '../../../../tsc/src/reference/latest';"
+    - "import Icon from '@mapbox/mr-ui/icon';"
+    - "import Property from '../../components/style-spec/property.js';"
+    - "import Subtitle from '../../components/style-spec/subtitle.js';"
+    - "import Note from '@mapbox/dr-ui/note';"
+---
+
+<!--copyeditor disable basic-->
+<!--copyeditor disable js-->
+<!--copyeditor disable ios-->
+<!--copyeditor disable macos-->
+<!--copyeditor disable all-of-->
+
+Some style properties are no longer the preferred method of accomplishing a particular styling goal. While they are still supported, they will eventually be removed from the MapLibre Style Specification and it is not recommended to use them in new styles. The following is provided as reference for users who need to continue maintaining legacy styles while transitioning to preferred style properties.
```

Comment:
I'm not sure we fully stand behind this remark.
So, I'm not sure how valuable this page is... I would consider removing it.